holidays/countries/taiwan.py

```diff
@@ -82,20 +87,56 @@ def __init__(self, *args, **kwargs):
     def _populate_observed(
         self, dts: set[date], rule: ObservedRule = None, since: int = 2015
     ) -> None:
+        """
+        Taiwan's General Observance Rule first started in 2015 as per
+        decreed in 內政部台內民字第1030182404號令.
+
+        Children's Day Special Observance Rule first started in 2012 as per
+        decreed in 內政部台內民字第1010307327號令 - as this doesn't affect 2011 ones,
+        as such the logic is simplified.
+        """
         if self._year < since:
             return None
 
+        # Children's Day.
         childrens_day = self.tr("兒童節")
         for dt in sorted(dts):
             names = self.get_list(dt)
             for name in names:
                 self._add_observed(
                     dt,
                     name,
-                    # Children's Day falls on the same day as Tomb Sweeping Day.
+                    # Children's Day falls on the same day as Tomb-Sweeping Day.
                     CHILDRENS_DAY_RULE if name == childrens_day and len(names) > 1 else rule,
                 )
 
+    def _populate_school_holidays(self):
```

Comment:
It's better to leave `_populate_public_holidays()` first, as in other countries.

Reply:
Fixed.